src/FoodStorage/foodStorage.py

```diff
@@ -1,16 +1,30 @@
+from src.Constants.constants import \
+    ADD_FOODS_SUCCESS_MESSAGE, \
+    EMPTY_MESSAGE, \
+    FOOD_ABOUT_TO_EXPIRE_MESSAGE, \
+    FOOD_ABOUT_TO_EXPIRE_MESSAGE_SINGULAR, \
+    FOOD_EXPIRED_MESSAGE, \
+    FOOD_EXPIRED_MESSAGE_SINGULAR, \
+    FOOD_EXPIRED_TODAY_MESSAGE
+from datetime import date, timedelta
 from queue import PriorityQueue
 
 class FoodStorage:
-    def __init__(self):
+    def __init__(self, expiration_window=7):
```

Comment:
Why did you add `expiration_window=7`?

Reply:
The `expiration_window` represents the window for what foods we want to display a notice for that are about to expire. I thought a default of 7 would be good. Basically this means when we create a `FoodStorage` object, by default, we give notice of foods that are going to expire in 7 days.